java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/common/DbzSourceUtils.java

```diff
@@ -62,8 +62,10 @@ public static void createPostgresPublicationIfNeeded(
                             ValidatorUtils.getSql("postgres.publication_exist"))) {
                 stmt.setString(1, pubName);
                 var res = stmt.executeQuery();
+                // Note: the value returned here is `pubviaroot`, If there's more than one row, the
```

Comment:
The `pubviaroot` only exist in PG version >= 13, we should use a more general sql to check the existence of the publication.

Reply:
🤔OKK, will use another PR to refactor this.

Reply:
I think we can revert back to use `count(*)` to check the existence.

Reply:
Partitioned table support relies on `pubviaroot`. Simply reverting will break the partitioned table support.

Reply:
fixed in https://github.com/risingwavelabs/risingwave/pull/18648